luxonis_ml/embeddings/methods/duplicate.py

```diff
@@ -83,7 +78,7 @@ def _plot_kde(xs, s, density, maxima, minima):
     plt.show()
 
 
-def kde_peaks(data, bandwidth="scott", plot=False):
+def kde_peaks(data: np.ndarray, bandwidth: Union[str, float] = "scott", plot: bool = False) -> tuple[np.ndarray, np.ndarray, int, float]:
```

Comment:
AFAIK luxonis-ml has python3.8>= requirement and for python3.8 `tuple` type annotation should be switched with `Tuple` or it will throw an error

Reply:
In addition to this, it would be nice to create at least a very dummy test case for each added module that at least imports everything, so it catches errors like this.